src/Core/Services/CipherService.cs

```diff
@@ -829,6 +829,24 @@ public async Task RestoreWithServerAsync(string id)
             await ClearCacheAsync();
         }
 
+        public async Task<bool> VerifyOrganizationHasUnassignedItemsAsync()
+        {
+            var organizations = await _stateService.GetOrganizationsAsync();
```

Comment:
🤔 I'm not sure if it's possible for this to return 0 orgs and the next call returning true, I just noticed other clients aren't checking this.

Reply:
This is exactly why I had to [wait for synchronization to complete](https://github.com/bitwarden/mobile/pull/3150/files#diff-d0d4f4baeed96a7ebebe5105ff70eb1bfad44ff3c0637043ed0b26eb80361b42R725).
I could remove this alongside the sync wait but it will fire a request to the server for users that don't have organizations which IMO is a waste of resources and given that we're waiting for sync to finish we should have the latest organizations state when checking it.